Vagrantfile

```diff
@@ -59,6 +59,13 @@ Vagrant.configure(2) do |config|
           "master" => ["master"],
         }
       end
+      master.vm.provision "miq", type: "ansible" do |ansible|
+        ansible.playbook = "miq.yaml"
+        ansible.groups = {
+          "nodes" => ["node"],
```

Comment:
nodes is not really needed here but it does not hurt either.

Reply:
Will remove